pkg/registry/servicecatalog/binding/storage.go

```diff
@@ -94,12 +94,12 @@ func toSelectableFields(binding *servicecatalog.Binding) fields.Set {
 }
 
 // GetAttrs returns labels and fields of a given object for filtering purposes.
-func GetAttrs(obj runtime.Object) (labels.Set, fields.Set, error) {
+func GetAttrs(obj runtime.Object) (labels.Set, fields.Set, bool, error) {
```

Comment:
@pmorie GetAttrs is not documented upstream. I do not expect anyone to know what these args are. @cheftako

https://github.com/kubernetes/apiserver/blob/master/pkg/storage/selection_predicate.go#L25-L27
```
// AttrFunc returns label and field sets and the uninitialized flag for List or Watch to match.
// In any failure to parse given object, it returns error.
type AttrFunc func(obj runtime.Object) (labels.Set, fields.Set, bool, error)
```

https://github.com/kubernetes/apiserver/blob/master/pkg/storage/selection_predicate.go#L25-L27